grammar.js

```diff
@@ -1482,6 +1482,11 @@ module.exports = grammar({
       paren_list(field('parameter', $._expression)),
     ),
 
+    exists: $ => seq(
```

Comment:
Postgres seems to extend this syntax a bit https://www.postgresql.org/docs/current/functions-subquery.html

Does it make sense to add `not in`, `any`, `some`... as well?
`in` does work already, but only for lists and not for subqueries.